### DISCUSSION TITLE
Budibase constant errors

### DISCUSSION THREAD

**Discussion by PterahNova** [Budibase/budibase#7268] [Help]
Hi there. I am currently trying to setup the budibase all in one image on Unraid. The image installs fine with port and data directory. The webpage opens but when changing anything on the server or trying to preview app etc the page simply errors out. The docker log has many errors which I have attached. Any help would be appreciated :)
[Console log.txt](https://github.com/Budibase/budibase/files/9336134/Console.log.txt)

**Comment by shogunpurple** [comment]
Hi @PterahNova - from the logs, it seems like you have run out of space on disk:

```
Error: Storage resources are insufficient for the write operation .minio.sys/buckets/prod-budi-app-assets/.usage-cache.bin (cmd.InsufficientWriteQuorum)
```

It's possible the partitiion that deals with volumes etc in docker is full, you will need to clear that out.

**Reply by PterahNova** [reply]
Hi there thank you for the reply, my docker image is currently 20GB with 6GB Used. I increased its size to 25GB to be sure and am still having the same issue. Its weird.

**Reply by PterahNova** [reply]
I see that is part of the /data directory. That directory has 83.9GB free atm.

**Reply by PterahNova** [reply]
I ended up using docker compose, up and running :)
